src/exchangeRate.ts

```diff
@@ -0,0 +1,58 @@
+import { asNumber, asObject, asOptional, asString } from 'cleaners'
+
+import config from '../config.json'
+
+export interface CurrencyOption {
+  address: string
+  currencyName: string
+  coinCapCurrencyName: string
+}
+
+const asPriceUsd = asObject({
+  priceUsd: asString
+})
+
+const asCoinCapResponse = asObject({
+  data: asOptional(asPriceUsd),
+  error: asOptional(asString),
+  timestamp: asNumber
+})
+
+const uri: string = config.coinCapUri
+export const currencies: { [key: string]: CurrencyOption } = config.currencies
+
+export const fetchExchangeRates = async (): Promise<{
```

Comment:
This function should use the Rates server and not coincap.

Reply:
Updated per comment.